api/controllers/voters.js

```diff
@@ -126,6 +125,8 @@ VotersController.getVoters = async function(context, next) {
 			_.pick(voter, ['address', 'publicKey', 'balance'])
 		);
 
+		data.votes = voters.length;
```

Comment:
It should be a total number of voters, not just length of the currently returned set. For example, if we specify `limit=1` but delegate have a total of `8000` voters it should still return `8000` here. Will it work like this?

Reply:
oh! Got it. It won't. Gonna fix it.

Reply:
@4miners  That's why the test was testing `to.be.least`, right?

Reply:
@4miners Looks like the test was not reliable. We should also update the description of this field in `swagger.yml` to match the current behavior.

Reply:
Ok. I will update swagger as well.